samples/Exporters/Console/TestOTelShimWithConsoleExporter.cs

```diff
@@ -0,0 +1,56 @@
+﻿// <copyright file="TestOTelShimWithConsoleExporter.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+using OpenTelemetry.Trace.Configuration;
+
+namespace Samples
+{
+    internal class TestOTelShimWithConsoleExporter
+    {
+        internal static object Run(OpenTelemetryShimOptions options)
+        {
+            // Enable OpenTelemetry for the source "MyCompany.MyProduct.MyWebServer"
+            // and use Console exporter.
+            using var openTelemetry = OpenTelemetrySdk.EnableOpenTelemetry(
```

Comment:
perhaps this could be renamed to something like TracerProvider.Create(builder action)?

Reply:
Rename `OpenTelemetrySdk.EnableOpenTelemetry` -> `TracerProvider.Create`? I like it.

Reply:
Ok. Will make it part of my upcoming PRs